cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/UserTaskState.java

```diff
@@ -69,19 +90,34 @@ private void addJSONTask(List<Map<String, Object>> jsonUserTaskList,
     jsonUserTaskList.add(jsonObjectMap);
   }
 
-  private void addFilteredJSONTasks(List<Map<String, Object>> jsonUserTaskList,
-                                    List<UserTaskManager.UserTaskInfo> userTasks,
-                                    String status,
-                                    Set<UUID> requestedUserTaskIds) {
-    for (UserTaskManager.UserTaskInfo userTaskInfo : userTasks) {
-      if (requestedUserTaskIds == null || requestedUserTaskIds.isEmpty() || requestedUserTaskIds.contains(userTaskInfo.userTaskId())) {
-        addJSONTask(jsonUserTaskList, userTaskInfo, status);
-      }
+  private <T> Predicate<UserTaskManager.UserTaskInfo> checkInputFilter(Set<T> set) {
+    if (set == null || set.isEmpty()) {
+      return elem -> true;
+    } else {
+      return elem -> false;
     }
   }
 
-  private String getPlaintext(CruiseControlParameters parameters) {
+  private void addFilteredJSONTasks(List<UserTaskManager.UserTaskInfo> userTasksFinal,
```

Comment:
Would it be possible to document the filtering priority in the JavaDoc of this class or this method?